fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -9,8 +9,14 @@
 from sklearn.utils.validation import check_is_fitted
 from warnings import warn
 
-from ..utils._common import _get_soft_predictions
-from ..utils._input_validation import _validate_and_reformat_input
+# for me to get it to work now
+from utils._common import _get_soft_predictions
+from utils._input_validation import _validate_and_reformat_input
+
+# how it should be
+# from ..utils._common import _get_soft_predictions
+# from ..utils._input_validation import _validate_and_reformat_input
```

Comment:
Anything you want help with here, or do you know how to work with this? Feel free to ask!

Reply:
I was/am struggling with getting these relative imports to work for some reason, however at some point I figured I just change it to absolute imports as for writing the actual code it does not really matter how these things are imported. If you have a clear idea what might be going wrong here (I get this error: from ..utils._common import _get_soft_predictions
ImportError: attempted relative import beyond top-level package), I'm all ears, but as I said I can manage with these absolute imports if not.

Reply:
I think your syntax is correct, I was having similar issues at some point too! Perhaps you are running this script directly? If so then that file is not part of the fairlearn module but the file is __main__ itself instead and hence .. is above the top-level package (itself).

Otherwise, you might find an answer [here](https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time/14132912#14132912)

Reply:
I think @SeanMcCarren is right. I removed your import changes using the `..` and made one other import change (removing `postprocessing` in one import as mentioned in another comment) and everything worked fine. Perhaps try `python -m purest test/unit/postprocessing` from the repo root directory to confirm.

Reply:
It seems to work now, thx :)!

Reply:
Are we reverting this change or leaving as is?

Reply:
we revert the change (so go with relative import)